build.sbt

```diff
@@ -6,6 +6,7 @@ ThisBuild / organizationName := "tessellation"
 
 ThisBuild / evictionErrorLevel := Level.Warn
 ThisBuild / scalafixDependencies += Libraries.organizeImports
+ThisBuild / libraryDependencySchemes += "io.circe" %% "circe-core" % VersionScheme.Always
```

Comment:
This alleviates the warnings for `circe-core` `0.13.0` being evicted over `0.14.3`. I think we probably shouldn't merge it in the end as it's known that for `circe-magnolia-derivation` the `0.13.0` is binary compatible with `0.14.3`, but at the same time it's not compatible in general as circe core uses Early semver and `0.13.0` can be binary incompatible with `0.14.3`. I just wanted to present it as one of the options to fix that warning, but ultimately it would be better to wait for the `circe-magnolia-derivation` release with bumped up `circe-core` and accept the warning for now.